ibm/service/atracker/data_source_ibm_atracker_targets.go

```diff
@@ -343,6 +350,20 @@ func DataSourceIBMAtrackerTargetsTargetToMap(model *atrackerv2.Target) (map[stri
 		}
 		modelMap["logdna_endpoint"] = []map[string]interface{}{logdnaEndpointMap}
 	}
+	if model.LogdnaEndpoint != nil {
```

Comment:
I think this block duplicated 346 (1st block)and 353  (same block)

Reply:
Fixed it